File: dgt/multas_agente/src/main/java/com/ipartek/formacion/controller/pojo/Mensaje.java
package com.ipartek.formacion.controller.pojo;

public class Mensaje {
	
	public static final String TIPO_INFO = "info";
	public static final String TIPO_PRIMARY = "primary";
	public static final String TIPO_SUCCESS = "success";
	public static final String TIPO_DANGER = "danger";
	public static final String TIPO_WARNING = "warning";
	
	private String texto;
	private String tipo;
	
	public Mensaje() {
		super();
		this.texto = "Error Inexperado";
		this.tipo = TIPO_DANGER;
	}

	public Mensaje(String tipo, String texto) {
		this();
		this.texto = texto;
		this.tipo = tipo;
	}

	public String getTexto() {
		return texto;
	}

	public void setTexto(String texto) {
		this.texto = texto;
	}

	public String getTipo() {
		return tipo;
	}

	public void setTipo(String tipo) {
		this.tipo = tipo;
	}

	@Override
	public String toString() {
		return "Alerta [texto=" + texto + ", tipo=" + tipo + "]";
	}
	
}

File: angular/src/app/model/mensaje.ts
export class Mensaje {

    private _texto: string;
    private _color: string;
    static DANGER ='alert-danger';
    static INFO ='alert-info';
    static SUCCESS='alert-success';
    static WARNING='alert-warning';
  

    constructor(texto: string, color: string= Mensaje.DANGER) {
        this._texto = texto;
        this._color = color;
    }

    public get texto(): string {
        return this._texto;
    }
    public set texto(value: string) {
        this._texto = value;
    }
    public get color(): string {
        return this._color;
    }
    public set color(value: string) {
        this._color = value;
    }
}
